src/Loaders/LoadingSpinner.tsx

```diff
@@ -26,6 +26,9 @@ interface Props {
   size?: Size;
 }
 
+// Length of animation
+const DURATION = 750;
```

Comment:
NBD: I know that it's commonly held to be a good idea to store constants instead of using magic numbers; but I prefer to at least keep them in the same function so the function has fewer external dependencies, and just so I don't have to go looking around for things. Also, use jsdoc style comments so that we get hinting from vs code elsewhere (meaning `/** Length of animation */` instead of `// `)